src/pages/journal/journal.scss

```diff
@@ -10,6 +10,22 @@ page-journal {
   .refresh-button-icon {
     padding: 0 10px;
   }
+  #unauthenticated-mode-indicator {
+    text-align: center;
+    padding: 16px 0;
+    background: map-get($colors-gds, 'gds-red');
+    #unauth-text {
+      color: #FFFFFF;
+      font-family: SFProDisplay-Regular;
```

Comment:
This font looks a bit out of place, is it from the design?

Reply:
it is from the design.